plugins/catalog-react/alpha-api-report.md

```diff
@@ -26,51 +24,60 @@ export function createEntityCardExtension<
   };
   disabled?: boolean;
   inputs?: TInputs;
-  configSchema?: PortableSchema<TConfig>;
```

Comment:
Note: I'm using the config here, would you recommend I remove it for now?
https://github.com/backstage/backstage/pull/20720/files#diff-b5146cd23deb66381041855a8297ad170ab1d0eb49c81e4dfb97b2c4a197045f

Reply:
As we discussed, I'll remove the config from the referenced pull request for now.